### PR TITLE
* Changed some specs for the modal promp, checkboxes and radio buttons

### PR DESCRIPTION
**Pull request by sroy3**

### CODE REVIEW

File: scss/components/collapsible.scss [hunk 1]
```diff
@@ -20,7 +21,7 @@
 
     cursor: pointer;
 
-    &:focus, &:hover {
```
Comment (GermainBergeron) [inline scss/components/collapsible.scss]: No more style on hover?

Comment (sroy3) [inline scss/components/collapsible.scss]: UX team decided to remove it, but you made me doubt and read again. It also needs to be blue when expanded even if not focused. I'll need to change that.

Comment (sroy3) [inline scss/components/collapsible.scss]: Done

File: scss/variables.scss [hunk 2]
```diff
@@ -21,11 +21,11 @@ $button-confirm-icon-width: 15px;
 $button-confirm-margin-bottom: 20px;
 
 // Checkboxes and radios
-$checkbox-size: 16px;
-$checkbox-label-margin: 16px;
+$checkbox-size: 18px;
```
Comment (JPLachance) [inline scss/variables.scss]: Is this breaking stuff in the UI? Like group privileges table? Or the Salesforce form?

Comment (sroy3) [inline scss/variables.scss]: I re-aligned them.

**Pull request merged**